src/panels/config/automation/trigger/types/ha-automation-trigger-event.ts

```diff
@@ -38,9 +47,26 @@ export class HaEventTrigger extends LitElement implements TriggerElement {
         .defaultValue=${event_data}
         @value-changed=${this._dataChanged}
       ></ha-yaml-editor>
+      <ha-users-picker
+        .pickedUserLabel=${this.hass.localize(
+          "ui.panel.config.automation.editor.triggers.type.event.context_user_picked"
+        )}
+        .pickUserLabel=${this.hass.localize(
+          "ui.panel.config.automation.editor.triggers.type.event.context_user_pick"
+        )}
+        .hass=${this.hass}
+        .value=${this._wrapUsersInArray(context)}
```

Comment:
Why do we need to do this?

Reply:
the `user_id` constraint can contain either a single user id or an array of them. This normalizes it to an array. Is there a better way to do this?